docs/docs/pyproject.md

```diff
@@ -246,27 +246,26 @@ pgsql = ["psycopg2"]
 databases = ["mysqlclient", "psycopg2"]
 ```
 
-When installing packages with poetry, you can specify extras by using the `-E|--extras` option:
+When installing packages with Poetry, you can specify extras by using the `-E|--extras` option:
 
 ```bash
 poetry install --extras "mysql pgsql"
 poetry install -E mysql -E pgsql
 ```
 
-When installing or specifying poetry-built packages, the extras defined in this section can be activated 
+When installing or specifying Poetry-built packages, the extras defined in this section can be activated 
 as described in [PEP 508](https://www.python.org/dev/peps/pep-0508/#extras).
 
-For example, the dependencies required by the `databases` extra can be activated when installing the 
-package using `pip` as shown below.
+For example, when installing the package using `pip`, the dependencies required by 
+the `databases` extra can be installed as shown below.
 
 ```bash
 pip install awesome[databases]
 ```
 
 !!!note
 
-    The dependencies specified for each `extra` must be already defined as project dependencies.
```

Comment:
Did you mean to remove this sentence? I think it's better to leave it, but with the "be already" → "already be" swapped.

Reply:
Oops that slipped past me, haven't yet had my cup of tea for the day 😁